src/components/Tabs/TabList/TabList.module.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../../styles/themes";
+@import "../../../styles/typography";
+
+.tabsWrapper {
+  padding-bottom: 20px;
+}
+
+.tabsWrapper .tabsList {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  font-size: 0;
+  position: relative;
+  height: 40px;
+  outline: none;
+}
+
+.tabsWrapper .tabsList.sm {
+  height: 32px;
+}
+
+.tabsWrapper .tabsList.lg {
+  height: 48px;
+}
+
+.tabsWrapper.stretched .tabsList {
+  width: 100%;
+}
+
+.tabsWrapper.stretched .tabsList .tabWrapper {
+  width: 100%;
+}
+
+.tabsWrapper.stretched .tabsList .tabWrapper .tabInner {
+  width: 100%;
+}
+
+.tabsWrapper.disabled {
+  opacity: 0;
+  height: 0;
+  padding-bottom: 5px;
+  pointer-events: none;
+}
+
+.tabsWrapper.disabled .tabsList .tabWrapper .tabInner:hover {
```

Comment:
looks to me like a potential issue. tabWrapper and tabInner:hover are in different component and therefore a different scope. I think this will not work. Am I missing something?

Reply:
still wating for an answer. tab inner is out of scope and will contain different class name

Reply:
Fixed it with`:global`